tensorflow/lite/micro/kernels/reduce.cc

```diff
@@ -73,7 +122,14 @@ TfLiteStatus EvalMean(TfLiteContext* context, TfLiteNode* node) {
   TfLiteReducerParams* params =
       reinterpret_cast<TfLiteReducerParams*>(node->builtin_data);
 
-  int num_axis = static_cast<int>(NumElements(axis));
+  // Interpret an axis tensor with null dimensions as a scalar
+  int num_axis;
+  if (axis->dims == nullptr) {
+    num_axis = 1;
+  } else {
+    num_axis = static_cast<int>(NumElements(axis));
+  }
```

Comment:
num_axis and num_elements are basically the same. It would be good to consolidate into a common local variable.

I'd prefer num_elements but since it was already called num_axis, you can use that too.